front/src/routes/dashboard/style.css

```
@@ -752,18 +754,42 @@
-/* Tablet dock: frosted pill, active tab inverted */
+/* Tablet dock: frosted pill, active tab inverted.
+   The frost (background + backdrop blur) is painted by the ::before layer
+   below, NOT by the bar itself, on purpose: the "…" overflow menu is a DOM
+   descendant of the bar, and an element with backdrop-filter is a backdrop
+   root — per spec (and strictly in Chromium) the backdrop-filter of a
+   descendant then only samples what is painted INSIDE that ancestor, never
+   the page behind it. With the blur on the bar, the menu's own blur was a
+   silent no-op on desktop Chrome/Edge: the cards' text showed through its
+   78% white, crisp (forum feedback). Safari does not isolate, which is why
+   the same CSS looked opaque on iOS. */
 :global(.glass-theme) .dashboardTabs {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)